src/components/ChatEntry.js

```diff
@@ -1,22 +1,43 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp'
 
 const ChatEntry = (props) => {
+
+
+  const updateLikeButtonClick = () =>{
+    const updateChat ={
+      id : props.id,
+      sender : props.sender,
+      body : props.body,
+      liked : !props.liked, 
+      timeStamp : props.timeStamp,
+    }
+    props.onUpdate(updateChat);
+  }
+
+  const heartColor = props.liked ? '❤️': '🤍'; 
```

Comment:
👍 Nice calculation of the emoji to use based on the liked prop. We could even do this inline in the JSX, though I do like the clarity it brings to do it on its own and give the result a good name.